srcpkgs/python-openssl/template

```diff
@@ -1,29 +1,15 @@
 # Template file for 'python-openssl'
 pkgname=python-openssl
 version=20.0.1
-revision=2
+revision=3
```

Comment:
We shouldn't need this revision bump, the new version should have exactly the same content as the old one.